test/integration/collada_world_exporter.cc

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (C) 2021 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#include <gtest/gtest.h>
+
+#include <ignition/common/Console.hh>
+#include <ignition/common/Filesystem.hh>
+
+#include "ignition/gazebo/Server.hh"
+#include "ignition/gazebo/test_config.hh"
+
+#include "helpers/UniqueTestDirectoryEnv.hh"
+
+using namespace ignition;
+using namespace gazebo;
+
+/////////////////////////////////////////////////
+class ColladaWorldExporterFixture : public ::testing::Test
+{
+  public: void SetUp() override
+  {
+    ignition::common::Console::SetVerbosity(4);
+  }
+
+  public: void LoadWorld(const std::string &_path)
+  {
+    ServerConfig serverConfig;
+    serverConfig.SetResourceCache(test::UniqueTestDirectoryEnv::Path());
+    serverConfig.SetSdfFile(common::joinPaths(PROJECT_SOURCE_PATH, _path));
+
+    std::cout << "Loading: " << serverConfig.SdfFile() << std::endl;
+    this->server = std::make_unique<Server>(serverConfig);
+    EXPECT_FALSE(server->Running());
+  }
+
+  public: std::unique_ptr<Server> server;
+};
+
+/////////////////////////////////////////////////
+TEST_F(ColladaWorldExporterFixture, ExportWorld)
+{
+  this->LoadWorld("test/worlds/collada_world_exporter.sdf");
```

Comment:
We should use `common::joinPaths` here too.

Reply:
Added